### PR TITLE
123 unique function fixing

### LINKED ISSUE #123: Wrong Definition
https://github.com/30-seconds/30-seconds-of-python#filter_non_unique

Non unique definition is return "non-unique values in a list." but function content return only unique values.
It must change function content to:
```python
def filter_non_unique(lst):
  return [item for item in lst if lst.count(item) < 1]
```
or header name must change to "filter_unique".

### PR DESCRIPTION
**Pull request by Natgho**
add new 'filter_unique' function and definition
## Description

Resolves #123

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (non-breaking improvement of a snippet)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly. 
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: snippets/filter_non_unique.md [hunk 1]
```diff
@@ -5,13 +5,14 @@ tags: list,beginner
 
 Filters out the non-unique values in a list.
 
-Use list comprehension and `list.count()` to create a list containing only the unique values.
+Use list comprehension and `list.count()` to create a list containing only the non-unique values.
```
Comment (Chalarangelo) [inline snippets/filter_non_unique.md]:
```suggestion
Use list comprehension and `list.count()` to create a list containing only the unique values.
```

File: snippets/filter_non_unique.md [hunk 2]
```diff
@@ -5,13 +5,14 @@ tags: list,beginner
...
 
 ```py
 def filter_non_unique(lst):
-  return [item for item in lst if lst.count(item) == 1]
+  return [x for x in set(item for item in lst if lst.count(item) > 1)]
```
Comment (Chalarangelo) [inline snippets/filter_non_unique.md]:
```suggestion
  return [item for item in lst if lst.count(item) == 1]
```

File: snippets/filter_non_unique.md [hunk 3]
```diff
@@ -5,13 +5,14 @@ tags: list,beginner
...
 ```
 
 ```py
-filter_non_unique([1, 2, 2, 3, 4, 4, 5]) # [1, 3, 5]
+filter_non_unique([1, 2, 2, 3, 4, 4, 5]) # [2, 4]
```
Comment (Chalarangelo) [inline snippets/filter_non_unique.md]:
```suggestion
filter_non_unique([1, 2, 2, 3, 4, 4, 5]) # [1, 3, 5]
```

File: snippets/filter_unique.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: filter_unique
+tags: list,beginner
+---
+
+Filters out the unique values in a list.
+
+Use list comprehension and `list.count()` to create a list containing only the unique values.
```
Comment (Chalarangelo) [inline snippets/filter_unique.md]:
```suggestion
Use list comprehension and `list.count()` to create a list containing only the non-unique values.
```

File: snippets/filter_unique.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+```py
+def filter_unique(lst):
+  return [item for item in lst if lst.count(item) == 1]
```
Comment (Chalarangelo) [inline snippets/filter_unique.md]:
```suggestion
  return [x for x in set(item for item in lst if lst.count(item) > 1)]
```

File: snippets/filter_unique.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+```
+
+```py
+filter_unique([1, 2, 2, 3, 4, 4, 5]) # [1, 3, 5]
```
Comment (Chalarangelo) [inline snippets/filter_unique.md]:
```suggestion
filter_unique([1, 2, 2, 3, 4, 4, 5]) # [2, 4]
```

**Comment by Natgho** [comment]
I am sorry for lateness. I was a little busy, thank you for merging.

**Pull request merged**